components/counter/src/styles/counter-group.scss

```diff
@@ -24,3 +24,7 @@
     gap: var(--ds-size-100, $ds-size-100);
     
 }
+
+[auro-dropdown] {
+  display: block;
+}
```

Comment:
Make sure all files in every repo have an EOF empty line. You can use a setting in VS Code to make sure this is true automatically.